src/app/proto/clinic.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package clinic_data;
+
+message Workers {
```

Comment:
Minor detail: name Employees might be politically more correct.

Reply:
I will change it in next serialization part but noted :)